build_tools/cmake/build_all.sh

```diff
@@ -38,6 +45,7 @@ declare -a CMAKE_ARGS=(
   "-DCMAKE_INSTALL_PREFIX=$(realpath ${INSTALL_DIR})"
   "-DIREE_ENABLE_ASSERTIONS=${IREE_ENABLE_ASSERTIONS}"
 
+
   # Use `lld` for faster linking.
   "-DIREE_ENABLE_LLD=ON"
```

Comment:
Drop extra newline? 

(Also, we'll try replacing this with `CMAKE_LINKER_TYPE` soon: https://discord.com/channels/689900678990135345/689906000043573354/1276956140965724261, https://cmake.org/cmake/help/latest/variable/CMAKE_LINKER_TYPE.html)